scale-tests/run.sh

```diff
@@ -0,0 +1,434 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
```

Comment:
Should we also `set -x` to make it possible/easy to re-run failed steps manually?

Reply:
I found `set -x` to be too verbose given the script logging. `container_exec` logs all commands (with the timestamp) for example.